src/js/program/program-controller.js

```diff
@@ -163,8 +171,8 @@ class ProgramController extends Eventable {
      * @returns {void}
      */
     preloadVideo() {
-        const { backgroundMedia, mediaController } = this;
-        const media = mediaController || backgroundMedia;
+        const { background: { currentMedia: backgroundMedia }, mediaController } = this;
+        const media = mediaController || backgroundMedia.activeMedia;
```

Comment:
Can you comment on why we preload the background's  currentMedia's backgroundMedia's activeMedia?

And why wouldn't currentMedia  not === mediaController?

Reply:
Oops, a bit of a refactoring typo